etc-conf/zypper.conf

```diff
@@ -7,3 +7,9 @@ gpgcheck = 0
 
 # Enable/Disable autorefresh on all repositories
 autorefresh = 0
+
+# Set SSL verification method for gpgkey download. Valid values: "yes", "no", "host", "peer".
+# gpgkey-ssl-verify = 'yes'
+
+# Set SSL verification method for repository usage. Valid values: "yes", "no", "host", "peer".
+# repo-ssl-verify = 'host'
```

Comment:
Shouldn't be default value `yes`?

Reply:
The default was "host" before therefore I picked this one. See:

https://github.com/candlepin/subscription-manager/blob/a6d8bbcbdd2848e10d1cb6853589060d97090593/src/subscription_manager/repofile.py#L636